Formula/simutrans.rb

```diff
@@ -1,15 +1,15 @@
 class Simutrans < Formula
   desc "Transport simulator"
   homepage "https://www.simutrans.com/"
-  url "https://downloads.sourceforge.net/project/simutrans/simutrans/121-0/simutrans-src-121-0.zip"
-  version "121.0"
-  sha256 "69fd95989761a013729106b48135f772f59126398cd93ada072f963c4d1e86b8"
+  url "svn://servers.simutrans.org/simutrans/trunk/", revision: "9274"
```

Comment:
Is this change announced anywhere?

Reply:
This was recommended in the [development forums](https://forum.simutrans.com/index.php/topic,20399.msg191591.html).